.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### General information
+
+  * Minishift version: 
+  * Download from: github releases / brew / chocolatey
+  * OS: Linux / macOS / Windows
+  * OS version/distro: 
+  * Hypervisor: KVM / xhyve / Hyper-V / VirtualBox
+  * `minishift start --shwo-libmachine-logs -v5`
```

Comment:
Typo: `--show-libmachine-logs`

Reply:
This one was noted after commit ;-) https://github.com/gbraad/minishift/commit/a636f6d078ae6e3c1c6635f32390b010b32e9aeb#commitcomment-25034542. The idea to publish was to get team feedback... so far, not much.

Reply:
Ha, indeed it was -- missed that.

Personally, though, I think having an issue template is a good idea.